### ISSUE TITLE
function calls inside lambda is warned as B008

### ISSUE THREAD

**Issue by toslunar** [PyCQA/flake8-bugbear#242]
Since flake8-bugbear 22.3.23, B008 warns the code below , but the function calls inside lambda is not called at function definition time.  It seems #239 caused this regression.
```python
def foo(f=lambda x: print(x)):
    f(1)


foo()
```

**Comment by jpy-git** [comment]
Ah yep functions in a lambda aren't executed a definition time so will fix this 👍
